pwndbg/gdblib/kernel/__init__.py

```diff
@@ -135,6 +138,58 @@ def is_kaslr_enabled() -> bool:
     return "nokaslr" not in kcmdline()
 
 
+@pwndbg.lib.cache.cache_until("start")
+def kbase() -> int | None:
+    arch_name = pwndbg.gdblib.arch.name
+
+    address = 0
+
+    if arch_name == "x86-64":
+        address = get_idt_entries()[0].offset
+    elif arch_name == "aarch64":
+        address = pwndbg.gdblib.regs.vbar
+    else:
+        return None
+
+    mappings = pwndbg.gdblib.vmmap.get()
+    for mapping in mappings:
+        # TODO: Check alignment
+
+        # only search in kernel mappings:
+        # https://www.kernel.org/doc/html/v5.3/arm64/memory.html
+        if mapping.vaddr & (0xFFFF << 48) == 0:
+            continue
+
+        if not mapping.execute:
+            continue
+
+        if mapping.vaddr <= address <= mapping.vaddr + mapping.memsz:
```

Comment:
Is this a bug? `<=` vs `<` at the end

Can also probably be:
```suggestion
        if address in mapping:
```